livestyled/models/ticket.py

```diff
@@ -259,7 +262,8 @@ def create_new(
             venue: Venue or str or int or None = None,
             event: Event or str or int or None = None,
             currency: Currency or None = None,
-            external_card_ref=None
+            external_card_ref=None,
+            additional_fields=None,
```

Comment:
Trailing comma

Reply:
I added it so next time we add a field the change is one line instead of two